shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/container/config/SuiteProxyClusterContainerConfigurationFactory.java

```diff
@@ -15,27 +15,30 @@
  * limitations under the License.
  */
 
-package org.apache.shardingsphere.test.integration.env.container.atomic.adapter.config.impl;
+package org.apache.shardingsphere.test.integration.container.config;
 
 import org.apache.shardingsphere.infra.database.type.DatabaseType;
 import org.apache.shardingsphere.test.integration.env.container.atomic.adapter.config.AdaptorContainerConfiguration;
 
-import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
 
 /**
- * Proxy cluster container configuration.
+ * Suite proxy cluster container configuration creator.
  */
-public final class DefaultProxyClusterContainerConfiguration implements AdaptorContainerConfiguration {
+public final class SuiteProxyClusterContainerConfigurationFactory {
```

Comment:
Please add private no argument constructor to avoid static class initialize.

Reply:
all done